spyder/plugins/editor/plugin.py

```diff
@@ -761,12 +761,17 @@ def get_plugin_actions(self):
                 _("Show code style warnings (pep8)"),
                 'code_analysis/pep8', 'set_pep8_enabled')
 
+        show_docstring_warnings_action = self._create_checkable_action(
+            _("Show docstring style warnings"), 'show_docstring_warnings',
+            'set_docstring_warnings')
+
         self.checkable_actions = {
                 'blank_spaces': showblanks_action,
                 'scroll_past_end': scrollpastend_action,
                 'indent_guides': showindentguides_action,
                 'show_class_func_dropdown': show_classfunc_dropdown_action,
-                'code_analysis/pep8': showcode_analysis_pep8_action}
+                'code_analysis/pep8': showcode_analysis_pep8_action,
+                'show_docstring_warnings': show_docstring_warnings_action}

```

Comment:
Is there a reason code style analysis is called ``code_analysis/pep8`` while docstring style analysis is called ``show_docstring_warnings``? Seems rather inconsistent to me, but it may be too late/out of scope to change or there's other reasoning behind it.